exportcontrol/settings.py

```diff
@@ -62,7 +63,7 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.getenv("DB_NAME", default="postgres"),
+        "NAME": os.getenv("DB_NAME", default="exportcontrol"),
```

Comment:
Should be kept as `postgres`, see https://github.com/cdp-group4/exportcontrol/pull/25#issuecomment-1243551737.